src/main/java/org/jenkinsci/plugins/gpjenkins/GlobalizationPipelineBuilder.java

```
@@ -373,8 +367,8 @@ private String getLanguageId(String gpLanguageTag, String langIdStyle,
 	private void mergeTranslation(Bundle bundle, String language, ResourceType type,
 			FilePath srcFile, FilePath outFile) throws InterruptedException, IOException {
 		ResourceFilter filter = ResourceFilterFactory.get(type);
-		try (FileOutputStream fos = new FileOutputStream(outFile.toURI().getPath());
-				FileInputStream fis = new FileInputStream(srcFile.toURI().getPath())) {
+		try (OutputStream fos = outFile.write();
+				InputStream fis = srcFile.read()) {
 			filter.merge(fis, fos, language, bundle);
 		} catch (IOException e) {
 			throw new IOException("I/O error while merging the translated values to "
```

I just noticed this code eat an exception and create a new IOException. It's OK to wrap an exception caught here to add more context/message, but you should also include the original exception - i.e. throw new IOException("....", e);